src/App.js

```diff
@@ -1,25 +1,46 @@
-import logo from './logo.svg';
+import { Switch, Route } from 'react-router-dom'
+import React, { useState } from 'react'
+import Home from './Home'
+import Contacts from './Contacts'
+import ContactNew from './ContactNew'
+import ContactPage from './ContactPage'
+
 import './App.css';
 
-function App() {
+const App = () => {
+  const [contacts, setContacts] = useState([
+    { id: 1, name: "Buddy Tan-Smith", image_url: "https://drive.google.com/uc?id=1zYfdxUW34Ic4mJyxtwxQENxV9fCqwQDy", email: "buddytsmith@gmail.com", phone_number: "555-555-5555" },
+    { id: 2, name: "Keroppi", image_url: "https://drive.google.com/uc?id=18i0_5jMufvqbveckqzN6hDnhD5vvl5PS", email: "ker0ppi@gmail.com", phone_number: "222-222-2222" }
+    
+  ]
+  )
+
+  const addContact = (contact) => {
+    setContacts(contacts => {
+      return [...contacts, contact]
```

Comment:
This is the source of all your bugs. The default state of `contacts` is always going to be those first two. Your impulse on line 20 is good, but I'd put line 20 into a variable and then pass it to `setContacts`